lib/jsdom.js

```diff
@@ -353,6 +375,17 @@ function getConfigFromArguments(args) {
     config.url = toFileUrl(config.file);
   }
 
+  config.pool = typeof config.pool !== "undefined" ? config.pool : {
```

Comment:
I know this code is messy. But is there any way to avoid duplicating the defaulting code?